assets/app/views/_templateopt.html

```diff
@@ -0,0 +1,21 @@
+<div class="template-options gutter-bottom" ng-show="hasParameters">
+  <div class="flow">
+    <div class="flow-block">
+      <h2>Template options</h2>
+    </div>
+    <div class="flow-block right">
```

Comment:
Any reason not to always show in edit mode? What happens if they edit some, then flip back to generate?